src/components/shared/inputs/Pay/PayInputGroup.tsx

```diff
@@ -38,49 +38,58 @@ export default function PayInputGroup({
 
   const [payAmount, setPayAmount] = useState<string>('0')
   const [payInCurrency, setPayInCurrency] = useState<CurrencyOption>(ETH)
+  const [isErrorField, setIsErrorField] = useState<boolean>(false)
 
   const togglePayInCurrency = () => {
     const newPayInCurrency = payInCurrency === ETH ? USD : ETH
     setPayInCurrency(newPayInCurrency)
   }
 
   return (
-    <div
-      style={{
-        display: 'flex',
-        width: '100%',
-      }}
-    >
-      <div style={{ flex: 1, marginRight: 10 }}>
-        <FormattedNumberInput
-          placeholder="0"
-          onChange={val => {
-            setPayAmount(val ?? '0')
-          }}
-          value={payAmount}
-          min={0}
-          accessory={
-            <InputAccessoryButton
-              withArrow={true}
-              content={currencyMetadata[payInCurrency ?? ETH].name}
-              onClick={togglePayInCurrency}
-            />
-          }
-        />
-        <PayInputSubText
-          payInCurrency={payInCurrency ?? ETH}
-          amount={payAmount}
-          reservedRate={reservedRate}
-          weight={weight}
-          tokenSymbol={tokenSymbol}
-          tokenAddress={tokenAddress}
-          weightingFn={weightingFn}
-        />
-      </div>
+    <>
+      {isErrorField && (
+        <span style={{ color: 'red', fontSize: '0.7rem' }}>
+          Pay amount must be greater than 0.
```

Comment:
**Issue:** don't forget to mark any text you add to the UI for translations. Will need to wrap this in <Trans> tags. Then run `yarn i18n:extract`, and commit the new translation files in your PR